config/locales/en.yml

```diff
@@ -346,7 +346,12 @@ en:
         new:
           title: About the accredited provider
           hint: Tell candidates about the accredited provider. You could mention their academic specialities and achievements.
-          caption: *accredited_provider_caption
+          caption: "Add accredited provider - %{provider_name} (%{code})"
+        edit: 
+          title: About the accredited provider
+          hint: Tell candidates about the accredited provider. You could mention their academic specialities and achievements.
+          caption: "%{provider_name}"
+          updated: About the accredited provider updated
```

Comment:
Are you actually using any of these?

Reply:
![Screenshot 2023-05-19 at 14 46 00](https://github.com/DFE-Digital/publish-teacher-training/assets/4527528/1fcba42f-62c1-46a3-835b-2ab8b30210d0)

Yes, I think so.

Reply:
Whats with caption?